src/System.Private.CoreLib/src/System/Internal.cs

```diff
@@ -1,365 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-/*============================================================
-**
-** This file exists to contain miscellaneous module-level attributes
-** and other miscellaneous stuff.
-**
-**
-**
-===========================================================*/
-
-// Code in this file isn't actually executed
-#nullable disable
-#pragma warning disable CA2007 // ConfigureAwait
-#pragma warning disable IDE0060 // unused parameters
-
-using System;
-using System.Collections.Generic;
-using System.Reflection;
-using System.Runtime.InteropServices;
-using System.Runtime.CompilerServices;
-using System.StubHelpers;
-using System.Threading.Tasks;
-
-#if FEATURE_COMINTEROP
-using System.Runtime.InteropServices.WindowsRuntime;
-#endif // FEATURE_COMINTEROP
-
-[assembly: CLSCompliant(true)]
-[assembly: ComVisible(false)]
```

Comment:
Should we move the attributes someplace else?

Reply:
Ah, missed those.  I'll leave the file and just delete other gunk.